Makefile

```diff
@@ -118,6 +118,17 @@ validate:
 
 ci: validate test release
 
+vendor:
+	export GO111MODULE=on \
+	$(GO) mod tidy && \
+	$(GO) mod vendor && \
+	$(GO) mod verify
+
+vendor-verify: vendor
```

Comment:
this should be put to the `validate` target?

Reply:
Yes! Thanks for catching it. Updated the PR now.